databroker/tests/test_core.py

```diff
@@ -3,13 +3,13 @@
 from databroker.core import Header
 
 
-def test_header_dict_conformance():
+def test_header_dict_conformance(db):
```

Comment:
The test is busted because of `_name`. It looks like you have future plans for this test but I can't figure out what they are. What's the right "fix" here?